rosidl_parser/rosidl_parser/definition.py

```diff
@@ -157,7 +157,7 @@ class NamedType(AbstractNestableType):
 
     def __init__(self, name: str):
         """
-        Constructor.
+        Create an AbstractNestableType.
```

Comment:
`Create a NamedType.`?

Reply:
Whoops. I was clearly looking at the supertype. Fixed.